### ISSUE TITLE
Assertion violation at CoreSMTSolver.cc:1049

### ISSUE THREAD

**Issue by rainoftime** [usi-verification-and-security/opensmt#85]
Hi, for the following formula,

[1049.txt](https://github.com/usi-verification-and-security/opensmt/files/4432823/1049.txt)

opensmt (commit 04c7fa3) throws an assertion violation
~~~~
unsat
sat
sat
unsat
sat
sat
sat
sat
sat
opensmt: /home/opensmt/src/smtsolvers/CoreSMTSolver.cc:1049: void CoreSMTSolver::analyze(CRef, vec<Lit>&, int&): Assertion `pathC == 1 || confl != CRef_Undef' failed.
Aborted (core dumped)
~~~~

**Comment by blishko** [comment]
Hi @rainoftime,
Thanks again for reporting these issues!
We suspect that many of them has common cause.
Could you please wait with your next batch until we fix the ones currently reported?
It will save time for both you and us if we can avoid dealing with duplicates.
Thanks!

**Comment by rainoftime** [comment]
@blishko  Thanks for the reply! I just noticed that there is a branch named "fixing issues".

**Comment by blishko** [comment]
@rainoftime Yes, we are working on the first batch you reported. Please wait until we process the issues you have already reported and merge the fixes to the master. Then it makes sense to test again. Otherwise, I fear you will report a lot of duplicates.

**Comment by blishko** [comment]
fixed by #80, should work in current master (81c0131991126c1003070f83b04faaf894403ff7)
